flutter/lib/src/screenshot/sentry_screenshot_quality.dart

```diff
@@ -17,4 +17,30 @@ enum SentryScreenshotQuality {
         return 854;
     }
   }
+
+  int calculateHeight(int width, int height) {
```

Comment:
doesn't need to be exposed - this is an implementation detail (a.k.a `@internal`)